packages/sit-onyx/src/components/TestInput/TestInput.vue

```diff
@@ -71,29 +73,43 @@ const emit = defineEmits<{
   validityChange: [state: ValidityState];
 }>();
 
-const isTouched = ref(false);
+const { t } = injectI18n();
 
-const inputElement = ref<HTMLInputElement | null>(null);
+const value = computed({
```

Comment:
isn't there a best practice to order the code e.g. 
1. all refs
2. all computed
3. ...
or was just something my previous project liked? ^^

Reply:
In general, I'd say
1. types / props / emits
2. composables
3. refs
4. computed / functions

I moved it

Reply:
I added it to [📜 #6 ](https://github.com/SchwarzIT/onyx/wiki/Technical-Vision-&-Guidelines#component-implementation)

Reply:
maybe smth we can write a linter rule later ^^